### PR TITLE
[Owner Review] Removing Calibration attributes from NI-FGEN

### PR DESCRIPTION
**Pull request by ShantanuShinde**
### What does this Pull Request accomplish?

It has been decided to not support the calibration APIs other than `SelfCal` for gRPC. So, the attributes for calibration have been removed from gRPC as they wont be used

### Why should this Pull Request be merged?

Calibration attributes removed

### What testing has been done?

The build is running successfully

### CODE REVIEW

File: source/codegen/metadata/nifgen/attributes.py [hunk 1]
```diff
@@ -280,43 +280,6 @@
         'resettable': True,
         'type': 'ViInt32'
     },
-    1150223: {
```
Comment (doshirohan) [inline source/codegen/metadata/nifgen/attributes.py:283]: Document removal of these attributes to CHANGES.md

File: source/codegen/metadata/nifgen/attributes.py [hunk 2]
```diff
@@ -329,12 +292,6 @@
         'resettable': True,
         'type': 'ViReal64'
     },
-    1150232: {
```
Comment (doshirohan) [inline source/codegen/metadata/nifgen/attributes.py]: This is not a calibration attribute. Do not remove this.

**Review by doshirohan — APPROVED** [review]
Update description with screenshot of CHM which highlights the attributes being removed. That would make it clear how we identified which attributes to remove to the reviewers.

File: source/codegen/metadata/nifgen/attributes.py (hunk 2 above)
Comment (Akshata440) [reply source/codegen/metadata/nifgen/attributes.py]: Yes this is not under calibration in the chm, dont remove this

**Review by Akshata440 — APPROVED** [review]

File: source/codegen/metadata/nifgen/attributes.py (hunk 1 above)
Comment (ShantanuShinde) [reply source/codegen/metadata/nifgen/attributes.py:283]: added the changes to the file

File: source/codegen/metadata/nifgen/attributes.py (hunk 2 above)
Comment (ShantanuShinde) [reply source/codegen/metadata/nifgen/attributes.py]: reverted that attribute back

**Review by ganeshnie — APPROVED** [review]

**Review by jonlugoatx — APPROVED** [review]

**Pull request merged**